README

```diff
@@ -134,23 +134,14 @@ CD-ROM: My CD-ROM doesn't work.
 
     In some cases you might want to use a different CD-ROM interface,
     for example if CD audio does not work:
-      To enable SDL-support (does not include low-level CD access!):
-        - mount d f:\ -t cdrom -usecd 0 -noioctl
-      To enable ioctl access using digital audio extraction for CD audio
-      (Windows-only, useful for Vista):
-        - mount d f:\ -t cdrom -ioctl_dx
-      To enable ioctl access using MCI for CD audio (Windows-only):
-        - mount d f:\ -t cdrom -ioctl_mci
-      To force ioctl-only access (Windows-only):
-        - mount d f:\ -t cdrom -ioctl_dio
+      - mount d f:\ -t cdrom -usecd 0 -noioctl
 
```

Comment:
The only remaining mount option is to try via SDL, so suggest rewording it a bit - 
---
You can also append `-usecd 0 -noioctl` to access the CDROM via SDL, if the above doesn't work.
For example:
- `mount d f:\ -t cdrom -usecd 0 -noioctl`
- `mount d /media/cdrom -t cdrom -usecd 0 -noioctl`

Reply:
Under Linux `-ioctl` and `-noioctl` work the same - both trigger "Ioctl" interface, which in case of Linux inherits from SDL interface (and not the pure abstract class). The only difference is, that when `-usecd` works only when `-noioctl` is passed (which is a bug, I presume).

Temporarily, this line works also on non-Linux systems, so... I think I'll leave it as it is - it will be removed very soon anyway.